assets/sass/components/_icons.scss

```diff
@@ -55,3 +55,16 @@ $spinnerColour: get-color('brand', 'primary');
         transform: rotate(360deg);
     }
 }
+
+.icon--download-btn {
```

Comment:
Should this be scoped more? Should _all_ download buttons be positioned this way or just for this use case? Same for the icon-tick styles.

Reply:
I think so, until they get used anywhere else? These styles are to basically make the icons usable, eg. align them optically and size them correctly.